consensus/src/dag/types.rs

```diff
@@ -418,6 +425,70 @@ impl Display for NodeId {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, CryptoHasher, Debug, PartialEq)]
+pub struct NodeMessage {
+    node: Node,
+    decoupled_payload: Option<DecoupledPayload>,
```

Comment:
Currently, PayloadInfo appears twice. In the future, I guess we will split this message?